### ISSUE TITLE
Segmentation prediction 

### ISSUE THREAD

**Issue by OverFitted** [ultralytics/yolov5#7021]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Is there any way to predict segmentation instead of bbox? 
I trained yolov5x6 on `class x1 y1 x2 y2 ... xn yn`-like dataset, but after predicting I get bbox not segments. 
```python
import torch

model = torch.hub.load('ultralytics/yolov5', 'custom', path="best.pt")
results = model("images/test/1060.png")
results.pandas().xyxy[0]
```
<html>
<body>

xmin | ymin | xmax | ymax | confidence | class | name
-- | -- | -- | -- | -- | -- | --
229.551865 | 248.660202 | 289.089813 | 291.088837 | 0.595772 | 2 | pituitary tumor

</body>
</html>

**Comment by glenn-jocher** [comment]
@OverFitted segmentation models are in our product pipeline for later this year. We can currently train classification models and detection models. Segmentation labels are currently used for better augmentation of box labels.

<img width="2163" alt="Screenshot 2021-11-22 at 18 26 43" src="https://user-images.githubusercontent.com/26833433/158890155-5f684eef-4cee-4024-93b5-74e6dd7de77c.png">
